client/my-sites/stats/features/modules/stats-top-posts/nav-tabs.tsx

```diff
@@ -1,25 +1,17 @@
 import page from '@automattic/calypso-router';
 import { TabPanel } from '@wordpress/components';
 import { useMemo } from 'react';
-import { useSelector } from 'react-redux';
 import {
 	getPathWithUpdatedQueryString,
 	trackStatsAnalyticsEvent,
 } from 'calypso/my-sites/stats/utils';
-import getEnvStatsFeatureSupportChecks from 'calypso/state/sites/selectors/get-env-stats-feature-supports';
-import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import useOptionLabels, {
 	MAIN_STAT_TYPE,
 	StatsModulePostsProps,
 	getValidQueryViewType,
 } from './use-option-labels';
 
 function NavTabs( { query }: StatsModulePostsProps ) {
-	const siteId = useSelector( getSelectedSiteId );
-	const { supportsArchiveStats } = useSelector( ( state: object ) =>
```

Comment:
The `supportsArchiveStats` here refers to the [compatibility support](https://github.com/Automattic/jetpack/pull/44028) with the [Jetpack version](https://github.com/Automattic/jetpack/pull/44052) rather than the feature flag.